modules/renter/hostdb/scan.go

```diff
@@ -46,14 +46,54 @@ var (
 // The gofunc is created inside of this function to eliminate the burden of
 // needing to remember to call 'go addHostToScanPool'.
 func (hdb *HostDB) scanHostEntry(entry *hostEntry) {
+	// Add the entry to a waitlist, then check if any thread is currently
+	// emptying the waitlist. If not, spawn a thread to empty the waitlist.
```

Comment:
instead of spawning a new thread as needed, why not just spawn a single thread at startup?